BLOG-POST-TEMPLATE.md

```diff
@@ -184,15 +186,25 @@ If you want an easier way to make tables in markdown, you can also try this
 
 ## Metadata
 
+We can use [OpenGraph meta tags](https://ogp.me/) for social sharing. Hugo has
+an embedded OpenGraph template:
+https://gohugo.io/templates/embedded/#open-graph.
+[Hugo OpenGraph embedded template source code](https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/opengraph.html).
+
 At this time, the blog generates metadata in the following ways:
 
-- the `title` parameter in the blog post head generates the `<title></title>`
+- The `title` parameter in the blog post head generates the `<title></title>`
   tag
-- [OpenGraph meta tags](https://ogp.me/) are generated for social sharing
+- The meta description in the blog post head is generated by the `description`
+  in the [blog post head](#blog-head). If no `description` is set in the front
+  matter of the post, there will not be a meta description tag.
+- [OpenGraph meta tags](https://ogp.me/):
   - [`og:title`](https://ogp.me/#metadata) is populated with the `title`
     parameter in the [blog post head](#blog-head)
-  - [`og:description`](https://ogp.me/#optional) is populated with the first
-    paragraph of the copy
+  - [`og:description`](https://ogp.me/#optional) is populated with the
+    `description` parameter in the [blog post head](#blog-head). If no
+    `description` is specified, this will default with the first paragraph of
+    the post.
   - [`og:type`](https://ogp.me/#metadata) is populated with the value
     `"article"`
   - [`og:url`](https://ogp.me/#metadata) is populated with the URL of the
```

Comment:
I can't comment on the line, but I believe `og:image` is populated with `images`, not `image`.